wordpress-phoenix-options-panel.php

```diff
@@ -1355,7 +1557,7 @@ public function get_html() {
 
 	public function echo_html() {
 		if ( ! empty( $this->filename ) && is_file( $this->filename ) ) {
-			return HTML::tag( 'li', [ 'class' => $this->get_clean_classname() ], file_get_contents( $this->filename ) );
+			return HTML::tag( 'li', [ 'class' => $this->get_clean_classname() ], wpcom_vip_file_get_contents( $this->filename ) );
```

Comment:
@debaryatimeinc depending on what @0aveRyan want's, we may have to do a condition here
if ( function_exists( 'wpcom_vip_file_get_contents' ) {
    // use wpcom_vip_file_get_contents
} else {
    // use file_get_contents // phpcs ignore line
}